index.js

```diff
@@ -0,0 +1,50 @@
+import Book from './modules/book.js';
+import BookCollection from './modules/bookCollection.js';
+import { addBookToLS, removeBookLS } from './modules/localStorage.js';
+import handleLinkClick from './modules/navbarLinks.js';
+import clearFormFields from './modules/clearForm.js';
+import getDate from './modules/date.js';
+
+// Display books on page load
+window.addEventListener('DOMContentLoaded', BookCollection.bookDisplay);
+
+// Add Book Event
+const formEl = document.querySelector('#book-form');
+formEl.addEventListener('click', (e) => {
+  e.preventDefault();
+  const title = document.querySelector('#book-name').value;
+  const author = document.querySelector('#book-author').value;
+
+  if (title !== '' && author !== '') {
+    if (e.target.classList.contains('addButton')) {
+      const newBook = new Book(title, author);
+      newBook.id = BookCollection.list.length + 1;
+      BookCollection.addBookUI(newBook);
+      addBookToLS(newBook); // Add book to booklist in localStorage
+      clearFormFields();
+      formEl.submit();
+    }
+  }
+});
+
+// Remove Book from UI, localStorage and arrayList
+const bookContainer = document.querySelector('.booksContainer');
+bookContainer.addEventListener('click', (e) => {
+  if (e.target.classList.contains('remove-btn')) {
+    const bookId = e.target.parentElement.id; // Card holder ID traversed here
+    BookCollection.removeBook(bookId);
+    removeBookLS(bookId);
+    e.target.parentElement.remove();
+  }
```

Comment:
- [ ] At the moment, the remove button is not working at all. Kindly ensure that the book is removed from the DOM and localStorage when the remove button is clicked

Reply:
Thank you for the review. I have checked my code and even live demo, its working very well,....I can remove the book both from the DOM and in Local Storage. Kindly help review from your end.

Reply:
@AnselemOdims Could it be your browser? How do I solve this issue?